app/js/reducers/fetchOrdersReducer.js

```diff
@@ -0,0 +1,50 @@
+const initialState = {
+  orders: {},
+  errorMessage: '',
+  status: {
+    fetched: false,
+    error: false,
+    loading: false,
+  },
+};
+
+const fetchOrdersReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case 'FETCH_ORDERS_SUCCESS':
+      return {
+        ...state,
+        orders: action.payload.data,
```

Comment:
I don't think it would be accessible as `action.payload.data` if it's a `SUCCESS` cos the responseMiddleware will format it as `action.data`

Reply:
On the contrary, it works fine as is, also my test breaks when I implement your changes and returns undefined.
But it works fine as is, please help me investigate if you still feel, that should not be the expected result.

Reply:
yeah, so it will work fine cos the payload would still be in action anyways but `action.data` is still the same thing as `action.payload.data`, for your test breaking, you would need to mock your response as `action.data` and get rid of the payload bit.

try logging action in the reducers and see what gives.

Reply:
Fixed, thanks.